.env2

```diff
@@ -0,0 +1,11 @@
+PG_USER=postgres
+PG_PASS=postgres
+PG_DB=wondertix
+PRIVATE_STRIPE_KEY=sk_test_51Md3wRFqQCFQg87vxBWXMv1pcykIoFjWWlB16jm69QZOUoyTpCqHd2GvotDSgKC2fIeRpse7mVhGz5p2vXgB7LxS00WSrN1SvO
```

Comment:
Adding private keys to a repo is a security issue.

1. Delete the commit(s) that added/edited this and force push it so that the key is not in the history.
2. Delete the key in stripe so it no longer has access
3. Generate a new key
4. We need to notify the client as well unless this is a key you generated on your own test account.